packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -197,10 +197,21 @@ function buildInstallCommandAndArgs(useYarn, saveDev = false) {
   return { command, args };
 }
 
-function sortScripts(scripts){
+function setupEnvars(appPath) {
+  osUtils.runExternalCommandSync('npx', ['@fs/fr-cli', 'env', 'local']);
+  const envPath = path.join(appPath, '.env');
+  let envFile = fs.readFileSync(envPath, 'UTF8');
+
+  envFile += `\nSKIP_PREFLIGHT_CHECK=true`;
```

Comment:
what is this for?

Reply:
this is to skip checking the dependencies against the whitelist of react-script's version of dependencies.

They do this so you can't upgrade the dependencies that any of react-scripts is using. Like they have a webpack.config.js file that was built and working with webpack fooVersion. If I update webpack to barVersion, there is no guarantee that webpack barVersion is compatible with their webpack.config written for fooVersion. So facebook has a preflight_check and won't even let you run the app if your dependencies aren't the same version as the dependencies in react-scripts.

Putting this flag on lets us skip that check.